### ISSUE TITLE
Penguins Eggs Arch (from archiso) Cannot find in ISO image: -boot_image ... bin_path='/isolinux/isolinux.bin'

### ISSUE THREAD

**Issue by codydafoxie** [pieroproietti/penguins-eggs-legacy#395]
Steps to reproduce:

install arch linux from archiso and archinstall (real hardware)
install eggs
customize the OS (DE: gnome)

`sudo eggs config`
`sudo eggs dad`
`sudo eggs produce -f -v --release` (throws the same error without any args or max compression)

error:

```
eggs >>> creating filesystem.squashfs on ISO/live.
mksquashfs /home/eggs/.mnt/filesystem.squashfs /home/eggs/.mnt/iso/live/filesystem.squashfs -comp xz -b 1M -wildcards -ef /etc/penguins-eggs.d/exclude.list -e 'boot/efi/EFI' 'etc/fstab' 'etc/mtab' 'etc/udev/rules.d/70-persistent-cd.rules' 'etc/udev/rules.d/70-persistent-net.rules' 'root/*' 'root/.*' 'home/eggs/' 
Parallel mksquashfs: Using 8 processors
Creating 4.0 filesystem on /home/eggs/.mnt/iso/live/filesystem.squashfs, block size 1048576.
[================================================================================================================/ ] 205164/205360  99%
Unrecognised xattr prefix system.posix_acl_access
[================================================================================================================| ] 205243/205360  99%
Unrecognised xattr prefix system.posix_acl_access
[================================================================================================================/ ] 205340/205360  99%
Unrecognised xattr prefix system.posix_acl_access
Unrecognised xattr prefix system.posix_acl_default
Unrecognised xattr prefix system.posix_acl_access
Unrecognised xattr prefix system.posix_acl_default
[=================================================================================================================/] 205360/205360 100%

Exportable Squashfs 4.0 filesystem, xz compressed, data block size 1048576
	compressed data, compressed metadata, compressed fragments,
	compressed xattrs, compressed ids
	duplicates are removed
Filesystem size 2410666.47 Kbytes (2354.17 Mbytes)
	33.45% of uncompressed filesystem size (7206963.04 Kbytes)
Inode table size 2167840 bytes (2117.03 Kbytes)
	21.45% of uncompressed inode table size (10108605 bytes)
Directory table size 3704468 bytes (3617.64 Kbytes)
	38.59% of uncompressed directory table size (9599662 bytes)
Xattr table size 358 bytes (0.35 Kbytes)
	36.76% of uncompressed xattr table size (974 bytes)
Number of duplicate files found 22565
Number of inodes 266389
Number of files 201760
Number of fragments 3010
Number of symbolic links 45544
Number of device nodes 0
Number of fifo nodes 0
Number of socket nodes 6
Number of directories 19079
Number of hard-links 37074
Number of ids (unique uids + gids) 21
Number of uids 11
	root (0)
	colord (968)
	gnome-remote-desktop (965)
	arch (1000)
	brltty (969)
	gdm (120)
	geoclue (966)
	uuidd (68)
	systemd-network (979)
	systemd-timesync (975)
	tss (974)
Number of gids 21
	root (0)
	brlapi (970)
	colord (968)
	gnome-remote-desktop (965)
	polkitd (102)
	arch (1000)
	users (984)
	ftp (11)
	groups (983)
	tty (5)
	dbus (81)
	games (50)
	brltty (969)
	gdm (120)
	geoclue (966)
	uuidd (68)
	systemd-network (979)
	systemd-timesync (975)
	tss (974)
	utmp (997)
	systemd-journal (982)
Ovary: uBindLiveFs
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/bin)
rm /home/eggs/.mnt/filesystem.squashfs/bin
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/boot -rf)
rm /home/eggs/.mnt/filesystem.squashfs/boot -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/dev -rf)
rm /home/eggs/.mnt/filesystem.squashfs/dev -rf
Ovary: mergedAndOverlay
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/etc)
umount /home/eggs/.mnt/filesystem.squashfs/etc
Ovary: rexec(umount /home/eggs/.overlay/lowerdir/etc)
umount /home/eggs/.overlay/lowerdir/etc
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/etc -rf)
rm /home/eggs/.mnt/filesystem.squashfs/etc -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/lib)
rm /home/eggs/.mnt/filesystem.squashfs/lib
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/lib64)
rm /home/eggs/.mnt/filesystem.squashfs/lib64
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/mnt -rf)
rm /home/eggs/.mnt/filesystem.squashfs/mnt -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/opt)
umount /home/eggs/.mnt/filesystem.squashfs/opt
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/opt -rf)
rm /home/eggs/.mnt/filesystem.squashfs/opt -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/proc -rf)
rm /home/eggs/.mnt/filesystem.squashfs/proc -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/root)
umount /home/eggs/.mnt/filesystem.squashfs/root
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/root -rf)
rm /home/eggs/.mnt/filesystem.squashfs/root -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/run -rf)
rm /home/eggs/.mnt/filesystem.squashfs/run -rf
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/sbin)
rm /home/eggs/.mnt/filesystem.squashfs/sbin
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/srv)
umount /home/eggs/.mnt/filesystem.squashfs/srv
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/srv -rf)
rm /home/eggs/.mnt/filesystem.squashfs/srv -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/sys -rf)
rm /home/eggs/.mnt/filesystem.squashfs/sys -rf
Ovary: mergedAndOverlay
Ovary: merged
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/tmp -rf)
rm /home/eggs/.mnt/filesystem.squashfs/tmp -rf
Ovary: mergedAndOverlay
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/usr)
umount /home/eggs/.mnt/filesystem.squashfs/usr
Ovary: rexec(umount /home/eggs/.overlay/lowerdir/usr)
umount /home/eggs/.overlay/lowerdir/usr
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/usr -rf)
rm /home/eggs/.mnt/filesystem.squashfs/usr -rf
Ovary: mergedAndOverlay
Ovary: rexec(umount /home/eggs/.mnt/filesystem.squashfs/var)
umount /home/eggs/.mnt/filesystem.squashfs/var
Ovary: rexec(umount /home/eggs/.overlay/lowerdir/var)
umount /home/eggs/.overlay/lowerdir/var
Ovary: rexec(rm /home/eggs/.mnt/filesystem.squashfs/var -rf)
rm /home/eggs/.mnt/filesystem.squashfs/var -rf
Ovary: xorrisoCommand
Ovary: makeDotDisk
mkdir /home/eggs/.mnt/iso/arch/x86_64/airootfs/x86_64 -p
Ovary: makeIso
xorriso -as mkisofs -J -joliet-long -l -iso-level 3 -isohybrid-mbr /usr/lib/syslinux/bios/isohdpfx.bin -partition_offset 16 -V arch -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table -eltorito-alt-boot -e boot/grub/efiboot.img -isohybrid-gpt-basdat -no-emul-boot -o /home/eggs/.mnt/archarch_amd64_2024-09-22_0624.iso /home/eggs/.mnt/iso/
xorriso 1.5.6 : RockRidge filesystem manipulator, libburnia project.

Drive current: -outdev 'stdio:/home/eggs/.mnt/archarch_amd64_2024-09-22_0624.iso'
Media current: stdio file, overwriteable
Media status : is blank
Media summary: 0 sessions, 0 data blocks, 0 data, 50.7g free
xorriso : WARNING : -volid text does not comply to ISO 9660 / ECMA 119 rules
Added to ISO image: directory '/'='/home/eggs/.mnt/iso'
xorriso : UPDATE :     313 files added in 1 seconds
xorriso : FAILURE : Cannot find in ISO image: -boot_image ... bin_path='/isolinux/isolinux.bin'
xorriso : UPDATE :     313 files added in 1 seconds
xorriso : aborting : -abort_on 'FAILURE' encountered 'FAILURE'
```

**Comment by codydafoxie** [comment]
`eggs love` returns the same error.

**Comment by pieroproietti** [comment]
Yes, I was just looking that this afternoon, and try to put in order.

this error arose from an addition I made for opensuse, I realized that opensuse did not work with the syslinux files internal to it but worked with the original files.
That said, realizing that it was the same for the other distributions as well, I added the original syslynux files in eggs. 

However, in Arch I forgot to add the directory containing them in the package build, it happens....

Now I've fixed it, but something is still bothering....
